extensions/amp-analytics/0.1/events.js

```diff
@@ -389,6 +395,43 @@ export class VisibilityTracker extends EventTracker {
   }
 
   /**
+   * @param {string=} waitForSpec
+   * @param {string=} selector
+   * @param {Element=} element
+   * @return {?Promise}
+   * @visibleForTesting
+   */
+  getReadyPromise(waitForSpec, selector, element) {
+    if (waitForSpec == WAITFOR_EVENTS_LIST.NONE || !selector) {
```

Comment:
I'd rather we resolve the default value and carefully document it as a first step.